twilight-model/src/guild/mod.rs

```diff
@@ -753,7 +750,7 @@ impl<'de> Deserialize<'de> for Guild {
                 }
 
                 for presence in &mut presences {
```

Comment:
I'm aware @vilgotf suggested this entire manual deserializer be removed, however it seems to inject `guild_id`s in channels and presences. If we were to remove this they would be set to `None` is this behavior desired?

Reply:
On second thought, I can probably just inject without the deserializer